src/pages/smart-contracts/clarity-values-guide.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Clarity Values Guide
+description: Learn how to deal with Clarity Values in JavaScript.
+tags:
+  - tutorial
```

Comment:
because this is not in the tutorial section of the smart contract chapter, we should remove the tutorial tag here